docs/pages/versions/unversioned/sdk/notifications.md

```diff
@@ -422,21 +422,65 @@ import React from 'react';
 import { Linking } from 'react-native';
 import * as Notifications from 'expo-notifications';
 
-export default function Container({ navigation }) {
+export default function Container() {
   React.useEffect(() => {
     const subscription = Notifications.addNotificationResponseReceivedListener(response => {
       const url = response.notification.request.content.data.url;
       Linking.openUrl(url);
     });
     return () => subscription.remove();
-  }, [navigation]);
+  }, []);
 
   return (
     // Your app content
   );
 }
 ```
 
+Handling push notifications with React Navigation
```

Comment:
I feel like this should be a header, so we can link to it easily (and it's more noticeable)